src/msg/async/net_handler.cc

```diff
@@ -44,7 +44,7 @@ int NetHandler::create_socket(int domain, bool reuse_addr)
   /* Make sure connection-intensive things like the benchmark
    * will be able to close/open sockets a zillion of times */
   if (reuse_addr) {
-    if (::setsockopt(s, SOL_SOCKET, SO_REUSEADDR, &on, sizeof(on)) == -1) {
+    if (::setsockopt(s, SOL_SOCKET, SO_REUSEADDR, nullptr, 0) == -1) {
```

Comment:
@wjwithagen sorry, i was wrong. we do need to pass a non-zero variable to `setsockopt()` here.